pkg/cmd/personal-token/create/create.go

```diff
@@ -0,0 +1,98 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/MakeNowJust/heredoc"
+
+	msg "github.com/aziontech/azion-cli/messages/personal-token"
+	api "github.com/aziontech/azion-cli/pkg/api/personal_token"
+
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/aziontech/azion-cli/utils"
+	"github.com/spf13/cobra"
+)
+
+type Fields struct {
+	Name        string
+	ExpiresAt   string
+	Description string
+	Path        string
+}
+
+func NewCmd(f *cmdutil.Factory) *cobra.Command {
+	fields := &Fields{}
+
+	cmd := &cobra.Command{
+		Use:           msg.CreateUsage,
+		Short:         msg.CreateShortDescription,
+		Long:          msg.CreateLongDescription,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		Example: heredoc.Doc(`
+        $ azion personal_token create --name "ranking of kings" --expiration "9m" 
+        $ azion personal_token create --name "sakura" --expiration "9m" 
+        $ azion personal_token create --name "luffy biruta" --expiration "9m" --description "gear five"
+        $ azion personal_token create --in "create.json"
+        `),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			request := api.Request{}
+
+			if utils.IsEmpty(fields.Name) || utils.IsEmpty(fields.ExpiresAt) {
```

Comment:
Is it worth creating an utils functions when c.Flags().Changed("name") does the same?

Reply:
It brakes the checking of the value and not the flag, so it even brakes the case when it enters invalid values in the struct or enters without the required fields